docker-compose.yml

```diff
@@ -12,12 +13,17 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
+      - HMPPS_SQS_LOCALSTACK_URL=http://hpfnm-localstack:4566
+      - API_BASE_URL_OAUTH=http://hpfnm-hmpps-auth:8080/auth
+      - API_BASE_URL_NOMIS=http://hpfnm-hmpps-nomis-prisoner-api:8080
+      - API_BASE_URL_VISITS=http://hpfnm-hmpps-visits-api:8080
+      - API_BASE_URL_VISIT_MAPPING=http://hpfnm-hmpps-nomis-visit-mapping-api:8080
 
   hmpps-auth:
     image: quay.io/hmpps/hmpps-auth:latest
     networks:
       - hmpps
-    container_name: hmpps-auth
+    container_name: hpfnm-hmpps-auth
```

Comment:
decided to rename all containers so that don't clash with other docker compose, and have to delete the "other" containers each time you switch projects